src/App.js

```diff
@@ -3,13 +3,15 @@ import "./styles.css";
 import { Modal } from "./modal/modal";
 import {days as dayKeys, fullDays} from './helper.js';
 import TaskCard from "./task-card.component";
-import useGetTasks from "./use-get-tasks";
+import {useGetTasks, useGetTaskDates} from './resource-hooks';
```

Comment:
two custom hooks in one file `resource-hooks` used for getting tasks and taskDates